syntax = "proto3";

message DataBaseReq {
    uint64 msg_id = 1;
	uint32 cmd_op = 2;
	bytes key = 3;
	bytes val = 4;
}

message DataBaseAck {
    uint64 msg_id = 1;
	uint32 query_state = 2;
	bytes result_str = 3;
}